pkg/rekor/identity.go

```diff
@@ -85,6 +91,9 @@ type MonitoredValues struct {
 	// FulcioExtensions contains all OID extensions currently supported by Fulcio
 	// each extension has a list of values to match on, ex. `build-signer-uri`
 	FulcioExtensions FulcioExtensions `yaml:"fulcioExtensions"`
+	// CustomOIDs contains a list of custom OID extension fields, represented in dot notation
+	// and associated values to match on. Used for input via CLI or GitHub Actions.
```

Comment:
nit: We can omit in the comment how it’ll be used, since this could also be used as an API if someone wants to write their own tool that uses this code.